keras_nlp/src/models/mix_transformer/mix_transformer_backbone_test.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+from keras import models
+
+from keras_nlp.src.models.mix_transformer.mix_transformer_backbone import (
+    MiTBackbone,
+)
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class MiTBackboneTest(TestCase):
+    def setUp(self):
+        self.init_kwargs = {
+            "depths": [2, 2, 2, 2],
+            "include_rescaling": True,
+            "input_image_shape": (224, 224, 3),
```

Comment:
Reduced it to 64

Reply:
lets keep them as small as possible, maybe 16?

Reply:
16 would fail, I think 64 is the minimum it si required, else it would fail like https://colab.sandbox.google.com/gist/sachinprasadhs/09e6c824daa2d6a3093f9ca44673d9b2/backbine-test.ipynb

Reply:
Modified the testcase.